Fails to Install

```<!--
If this is a bug, please use the template below. If this is a question or general discussion topic, please start a conversation in our chat https://chat.pop-os.org/ or post on our subreddit https://reddit.com/r/pop_os - as those are the proper forums for that type of discussion.
-->

**(1) Issue/Bug Description:**
Fails to Install


**(2) Steps to reproduce (if you know):**
make local-install


**(3) Expected behavior:**
Clean Install


**(4) Distribution (run `cat /etc/os-release`):**
 NAME="Solus"
VERSION="4.1"
ID="solus"
VERSION_CODENAME=fortitude
VERSION_ID="4.1"
PRETTY_NAME="Solus 4.1 Fortitude"
ANSI_COLOR="1;34"
HOME_URL="https://getsol.us"
SUPPORT_URL="https://getsol.us/articles/contributing/getting-involved/en/"
BUG_REPORT_URL="https://dev.getsol.us/"



**(5) Gnome Shell version:**
GNOME Shell 3.38.1



**(6) Pop Shell version (run `apt policy pop-shell` or provide the latest commit if building locally):**
<!--
Use triple back ticks ```<value here>``` to paste the apt policy pop-shell value, so it formats nicely
Pop-shell does not exist on the system



**(7) Where was Pop Shell installed from:**
ee943b87bd00b2bb7b3ecf970f4f88741346982e

**(8) Monitor Setup (2 x 1080p, 4K, Primary(Horizontal), Secondary(Vertical), etc):**
1 Monitor 1080p Horizontal


**(9) Other Installed/Enabled Extensions:**
ls ~/.local/share/gnome-shell/extensions/
hidetopbar@mathieu.bidon.ca  pop-shell@system76.com


**(10) Other Notes:**
+ dconf write /org/gnome/mutter/workspaces-only-on-primary false
gnome-extensions enable "pop-shell@system76.com"
Extension “pop-shell@system76.com” does not exist
make: *** [Makefile:61: enable] Error 2

I installed a prior version of this on the same OS fine, Gnome shell version was 3.36.3 at that time, and the commit used rebuild.sh still, I don't know how to find the version```

Can confirm this issue on **Ubuntu 20.10** - **Gnome 3.38.1**. **Kernel: 5.8.0-31-generic**

**Error:** gnome-extensions enable "pop-shell@system76.com"
Extension “pop-shell@system76.com” does not exist
make: *** [Makefile:61: enable] Error 2

When run "make local-install"

Hoping someone comes up with a solution soon I love the pop-shell but can't migrate to pop-os.

I get a similar error in ubuntu 18.04

+ gnome-extensions list
+ scripts/configure.sh: 105: scripts/configure.sh: gnome-extensions: not foundgrep
 native-window
+ dconf write /org/gnome/mutter/workspaces-only-on-primary false
gnome-extensions enable "pop-shell@system76.com"
/bin/sh: 1: gnome-extensions: not found
Makefile:61: recipe for target 'enable' failed
make: *** [enable] Error 127

--- a/Makefile
+++ b/Makefile
@@ -73,7 +73,7 @@ disable:
 listen:
 	journalctl -o cat -n 0 -f "$$(which gnome-shell)" | grep -v warning
 
-local-install: depcheck compile install configure enable restart-shell
+local-install: depcheck compile install configure restart-shell enable
 
 install:
 	rm -rf $(INSTALLBASE)/$(INSTALLNAME)
@@ -92,6 +92,7 @@ restart-shell:
 	else \
 		gnome-session-quit --logout; \
 	fi
+	sleep 3
 
 update-repository:
 	git fetch origin

